docs/choreolib/auto-controller.md

```diff
@@ -0,0 +1,69 @@
+# Auto Controller
+
+Choreo trajectory samples contain a lot of information. Due to this complexity, the
+task of implementing how to follow these samples is left up to the user.
+
+The "Auto Controller" is a function that consumes a `Pose2d` as well as a sample (`SwerveSample` or `DifferentialSample` depending on drive base).
+The controller should capture your drive subsystem to directly control it.
+
+## Basic PID Controller
```

Comment:
This particular implementation is overkill. I think we should recommend a function in the drivebase subsystem, which captures the subsystem and PID controllers. This is how the CTRE examples work.